docs/crud/subscriptions.md

```diff
@@ -4,4 +4,75 @@ title: Subscriptions
 sidebar_label: Subscriptions
 ---
 
-## TODO
\ No newline at end of file
+Subscriptions are divided into three groups of changes: **Create**, **Update** and **Delete**. 
+
+- `new<Type>`: subscribe to the [**Create**](./mutations#create) mutation event.
+- `updated<Type>`: subscribe to the [**Update**](./mutations#update) mutation event.
+- `deleted<Type>`: subscribe to the [**Delete**](./mutations#delete) mutation event.
+
+Graphback generates a subscription handler and filter input for each CRUD mutation field to let you subscribe to mutation events and receive real-time updates.
+
+```graphql
+""" @model """
+type Note {
+  id: ID!
+  title: String!
+  likes: Int
+}
+
+// highlight-start
+input NoteSubscriptionFilter {
+  id: ID
+  title: String
+  likes: Int
+}
+
+type Subscription {
+  newNote(filter: NoteSubscriptionFilter): Note!
+  updatedNote(filter: NoteSubscriptionFilter): Note!
+  deletedNote(filter: NoteSubscriptionFilter): Note!
+}
+// highlight-end
+```
+
+:::caution
+Subscription filtering is not available yet and will be implemented in a future release.
```

Comment:
Added this from https://github.com/aerogear/graphback/issues/1273#issuecomment-655489593